taskt/taskt.csproj

```diff
@@ -847,6 +847,10 @@
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </Content>
     <None Include="Resources\command-dictionary.png" />
+    <None Include="Resources\command_folders.bmp" />
+    <None Include="Resources\command-folders.png" />
```

Comment:
Are these resources (`command_folders.bmp`, `command-folders.png`) similar? I could see that you used only `command_folders.bmp` in this update.

Reply:
They're different. I actually edited the original to look more like the other buttons but I guess I forgot to make the name different and the original wasn't overwritten because the file extensions are different.